src/Feature/Catalog/tests/Controllers/SearchControllerTests.cs

```diff
@@ -20,34 +20,32 @@ namespace Wooli.Feature.Catalog.Tests.Controllers
 
     using Foundation.Base.Models;
     using Foundation.Commerce.Models;
-    using Foundation.Commerce.Models.Entities;
     using Foundation.Commerce.Models.Entities.Search;
-    using Foundation.Commerce.Services.Catalog;
+    using Foundation.Commerce.Services.Search;
 
     using Mappers;
 
-    using Models.Requests;
+    using Models.Requests.Search;
 
     using NSubstitute;
 
     using Xunit;
 
     public class SearchControllerTests
     {
-        private readonly IProductSearchService productSearchService;
-
-        private readonly ICatalogEntityMapper catalogEntityMapper;
-
-        private readonly SearchController controller;
-
         public SearchControllerTests()
         {
             this.productSearchService = Substitute.For<IProductSearchService>();
-            this.catalogEntityMapper = Substitute.For<ICatalogEntityMapper>();
+            this.searchMapper = Substitute.For<ISearchMapper>();
 
-            this.controller = Substitute.For<SearchController>(this.productSearchService, this.catalogEntityMapper);
+            this.controller = Substitute.For<SearchController>(this.productSearchService, this.searchMapper);
         }
 
+        private readonly IProductSearchService productSearchService;
+
+        private readonly ISearchMapper searchMapper;
+
+        private readonly SearchController controller;
```

Comment:
Move `private readonly` fields before constructor

Reply:
Ok, will fix this todether with all unit tests TODOs